localpdb/utils/search_api/search.py

```diff
@@ -0,0 +1,114 @@
+import json
+import urllib
+from abc import abstractmethod
+
+import requests
+from queries import TerminalQuery
+from services import SeqmotifService, SequenceService, StructureService, StructMotifService
```

Comment:
Fix so that `localpdb.utils.search_api.search` can be imported outside of the directory.
```suggestion
from .queries import TerminalQuery
from .services import SeqmotifService, SequenceService, StructureService, StructMotifService
```

Reply:
Good idea :), changed